src/main/java/io/github/mfvanek/pg/model/IndexWithNulls.java

```diff
@@ -36,6 +37,26 @@ public String toString() {
                 '}';
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Please cover all new `equals` and `hashCode` methods with proper unit tests

Reply:
Added